frontend/src/components/EditProfile/EditProfile.js

```diff
@@ -0,0 +1,147 @@
+import Axios from 'axios'
```

Comment:
can you lowercase Axios? It's been lowercased on all other documents (you'd just need to change it here and on line 17)